backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2KeyController/KeyControllerWithNamespacesTest.kt

```diff
@@ -231,4 +234,10 @@ class KeyControllerWithNamespacesTest : ProjectAuthControllerTest("/v2/projects/
     projectFetched.useNamespaces = true
     projectService.save(projectFetched)
   }
+
+  private fun disableNamespaces() {
```

Comment:
Is this used?

Reply:
No, I thought I removed it, now it's really removed.